sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -94,6 +94,81 @@ public static void validateUpdateContent(Request contentRequestDto) {
               ERROR_CODE);
         }
       }
+      List<Map<String, Object>> assessmentData = (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.ASSESSMENT_EVENTS);
+      System.out.println(assessmentData);
+      if (null != assessmentData && !assessmentData.isEmpty()) {
```

Comment:
Use MapUtils.isEmpty(assessmentData)

Reply:
Used CollectionUtils.isEmpty(assessmentData) since it is of list type